GradleException: CAS Gradle Plugin: no build variants found for the module rtn-cas-ads. The plugin requires at least one build variant that uses in Application build.

I am adding CAS Android SDK to React Native Fabric module.
Following the manual I added `repositories` in `settings.gradle` and applied plugin to `build.gradle` in fabric module, then setting up `cas` plugin.

`mobile-app/android/settings.gradle`
```groovy
dependencyResolutionManagement {
  repositories {
    google()
    mavenCentral()
    maven {
      name = "IronSourceAdsRepo"
      url = "https://android-sdk.is.com/"
      content { it.includeGroup("com.ironsource.sdk") }
    }
...
```

`mobile-app/modules/RTNCasAds/android/build.gradle`
```groovy
plugins {
    id("com.cleveradssolutions.gradle-plugin") version "3.9.7.1"
}

cas {
    casId = "testId"
    useAdvertisingId = true
    includeOptimalAds = true
}
```

When I am trying to run or sync I have an error:
> Caused by: org.gradle.api.GradleException: CAS Gradle Plugin: no build variants found for the module rtn-cas-ads. The plugin requires at least one build variant that uses in Application build.
	at com.cleveradssolutions.gradleplugin.CASPlugin.handleAfterEvaluate(CASPlugin.kt:58)

Build variants `debug` and `release` are presented in `mobile-app/modules/RTNCasAds/android/build.gradle`.

There are full versions of settings and build gradle in my Fabric module:
[build.gradle.txt](https://github.com/user-attachments/files/18210438/build.gradle.txt)
[settings.gradle.txt](https://github.com/user-attachments/files/18210440/settings.gradle.txt)

![image](https://github.com/user-attachments/assets/5634b135-67a7-4704-95a0-0b4d5f6b091e)

--- a/build.gradle
+++ b/build.gradle
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.cas_version = '3.9.8'
+    ext.cas_version = '3.9.8.1'
 
     ext.kotlin_version = '1.8.22'
 

